TTHAnalysis/python/tools/higgsDiffRecoTTH.py

```diff
@@ -70,7 +72,10 @@ def beginFile(self, inputFile, outputFile, inputTree, wrappedOutputTree):
     def analyze(self, event):
         # Some useful input parameters
         year=getattr(event,"year")
-        btagvetoval= HiggsRecoTTHbtagwps["DeepFlav_%d_%s"%(year,self.btagDeepCSVveto)][1]
+        if self.btagDeepCSVveto in ['VL','L','M','T','VT']:
```

Comment:
Is this to set a numerical value by hand, different from the official points, to the btag veto value?